### PR TITLE
Remove jormun python 2.7

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: .github/workflows/build_dockers.yml
Comment (woshilapin) [inline .github/workflows/build_dockers.yml:1]: This file can probably be removed entirely.

File: debian/control [hunk 1]
```diff
@@ -69,21 +69,7 @@ Depends: python-psycopg2, python-protobuf
...
-    python-all
-X-Python-Version: >=2.7
-Architecture: all
-Maintainer: Vincent Lara <vincent.lara@canaltp.fr>
-Depends: python-protobuf, python-psycopg2, navitia-common (= ${binary:Version}), ${python:Depends}
-Section: devel
-Priority: optional
-Description: Navitia-Jormungandr
-
+    python module shared within module of navitia, mostlu and navitia-tyr
```
Comment (woshilapin) [inline debian/control]:
```suggestion
    python module shared within module of navitia, mostly navitia-tyr
```

File: debian/control [hunk 2]
```diff
@@ -69,21 +69,7 @@ Depends: python-psycopg2, python-protobuf
 Section: devel
 Priority: optional
 Description: navitia-common
-    python module shared within module of navitia, mostlu navitia-jormungandr and navitia-tyr
-
-Package: navitia-jormungandr
```
Comment (woshilapin) [inline debian/control:74]: I'm not sure if I understand exactly what's going on here. We remove a package dependency? If that's the case, why don't we need it anymore?

**Review by woshilapin — APPROVED** [review]
Nice to see all of that technical debt going away :rocket:

File: .github/workflows/build_dockers.yml
Comment (azime) [reply .github/workflows/build_dockers.yml:1]: :+1:

File: debian/control (hunk 1 above)
Comment (azime) [reply debian/control]: Done

File: debian/control (hunk 2 above)
Comment (azime) [reply debian/control:74]: we no longer make debian8 packages for Jormun

File: debian/control (hunk 1 above)
Comment (azime) [reply debian/control]: Done

**Review by woshilapin — APPROVED** [review]

**Review by pbench — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Review by pbench — APPROVED** [review]

**Pull request merged**